diff -Naur orig/bashline.c patched/bashline.c
--- orig/bashline.c	2014-02-10 01:56:58.000000000 +0100
+++ patched/bashline.c	2015-06-13 16:21:04.683158913 +0200
@@ -590,6 +590,9 @@
     posix_readline_initialize (1);
 #endif
 
+  /* Make sure editing of long lines is okay right after bash starts up. */
+  get_new_window_size (0, (int *)0, (int *)0);
+
   bash_readline_initialized = 1;
 }
 
diff -Naur orig/config-top.h patched/config-top.h
--- orig/config-top.h	2012-08-19 02:51:30.000000000 +0200
+++ patched/config-top.h	2015-06-13 16:25:52.076584020 +0200
@@ -122,7 +122,7 @@
 
 /* Define to 0 if you want the checkwinsize option off by default, 1 if you
    want it on. */
-#define CHECKWINSIZE_DEFAULT	0
+#define CHECKWINSIZE_DEFAULT	1
 
 /* Define to 1 if you want to optimize for sequential array assignment when
    using indexed arrays, 0 if you want bash-4.2 behavior, which favors
