diff -Naur orig/bashhist.c patched/bashhist.c
--- orig/bashhist.c	2012-06-07 18:52:31.000000000 +0200
+++ patched/bashhist.c	2015-06-13 16:31:26.578242724 +0200
@@ -278,7 +278,7 @@
      Note that the history file is automatically truncated to the
      size of HISTSIZE if the user does not explicitly set the size
      differently. */
-  set_if_not ("HISTSIZE", "500");
+  set_if_not ("HISTSIZE", "100000");
   sv_histsize ("HISTSIZE");
 
   set_if_not ("HISTFILESIZE", get_string_value ("HISTSIZE"));
