diff -Naur orig/lumina-desktop/Lumina-DE.desktop patched/lumina-desktop/Lumina-DE.desktop
--- orig/lumina-desktop/Lumina-DE.desktop	2015-10-23 20:08:10.000000000 +0200
+++ patched/lumina-desktop/Lumina-DE.desktop	2015-11-22 01:09:09.069797206 +0100
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Exec=Lumina-DE
 TryExec=Lumina-DE
-Icon=/usr/local/share/pixmaps/Lumina-DE.png
+Icon=Lumina-DE
 Type=Application
 Name=Lumina
 Name[de]=Lumina
diff -Naur orig/lumina-fm/lumina-fm.desktop patched/lumina-fm/lumina-fm.desktop
--- orig/lumina-fm/lumina-fm.desktop	2015-10-23 20:08:10.000000000 +0200
+++ patched/lumina-fm/lumina-fm.desktop	2015-11-22 01:09:09.069797206 +0100
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Exec=lumina-fm %F
-Icon=/usr/local/share/pixmaps/Insight-FileManager.png
+Icon=Insight-FileManager
 Terminal=false
 Type=Application
 StartupNotify=true
diff -Naur orig/lumina-info/lumina-info.desktop patched/lumina-info/lumina-info.desktop
--- orig/lumina-info/lumina-info.desktop	2015-10-23 20:08:10.000000000 +0200
+++ patched/lumina-info/lumina-info.desktop	2015-11-22 01:10:04.918074142 +0100
@@ -1,11 +1,10 @@
 [Desktop Entry]
 Exec=lumina-info
-Icon=/usr/local/share/pixmaps/Lumina-DE.png
+Icon=Lumina-DE
 Terminal=false
 Type=Application
 StartupNotify=true
 Categories=System;
-OnlyShowIn=Lumina
 Name=Desktop Information
 Comment=View information about the Lumina Desktop Environment
 
