diff -Naur orig/OS-detect.pri patched/OS-detect.pri
--- orig/OS-detect.pri	2016-02-10 21:41:00.000000000 +0100
+++ patched/OS-detect.pri	2016-02-13 00:54:48.328577440 +0100
@@ -85,7 +85,7 @@
   # If the detailed install variables are not set - create them from the general vars
   !defined(L_BINDIR){ L_BINDIR = $${PREFIX}/bin }
   !defined(L_LIBDIR){ L_LIBDIR = $${PREFIX}/lib }
-  !defined(L_ETCDIR){ L_ETCDIR = $${PREFIX}/etc }
+  !defined(L_ETCDIR){ L_ETCDIR = /etc }
   !defined(L_SHAREDIR){ L_SHAREDIR = $${PREFIX}/share }
   !defined(L_INCLUDEDIR){ L_INCLUDEDIR = $${PREFIX}/include }
   !defined(L_SESSDIR){ L_SESSDIR = $${L_SHAREDIR}/xsessions }
